src/figuresClasses.ts

```diff
@@ -2,10 +1,99 @@
+type Shape = string | 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
+
+export class Rectangle implements Figure {
+  shape: string = 'rectangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  private width: number;
+
+  private height: number;
+
+  constructor(color: 'red' | 'green' | 'blue', width: number, height: number) {
+    if (width <= 0 || height <= 0) {
+      throw new Error('Width and height must be greater than zero!');
+    }
+    this.color = color;
+    this.width = width;
+    this.height = height;
+  }
+
+  getArea(): number {
+    const area = this.width * this.height;
+
+    return Math.floor(area * 100) / 100;
+  }
+}
+
+export class Circle implements Figure {
+  shape: string = 'circle';
```

Comment:
[CODE STYLE] - Just like in the 'Rectangle' class, use the enum for 'shape' with capitalized constant names for the 'shape' property in 'Circle'.